Pages/checkoutPage.js

```diff
@@ -28,6 +28,13 @@ class CheckoutPage {
     async returnToCartPage() {
         await t.click(this.cancelBttn);
     }
+
+    async checkoutMyInformation(myPersonalDetails) {
```

Comment:
the function is well done, but I would change the name of the parameter to something else, a more generic naming that is not the same as your myPersonalDetails object's name. maybe 'user'. This should help you understand better the function declaration vs function invocation.